modules/consensus/consensusset.go

```diff
@@ -50,6 +48,11 @@ type ConsensusSet struct {
 	// up loading blocks from memory.
 	verificationRigor verificationRigor
```

Comment:
We should be able to. I for now just to keep the volume of changes small, but as it currently stands it is never set as anything other than full, so removing it shouldn't pose a problem.